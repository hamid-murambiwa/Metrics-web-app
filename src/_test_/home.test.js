import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../redux/configureStore';
import Home from '../pages/home';

describe('Render components', () => {
  test('render HomePage', () => {
    const tree = renderer
      .create(<Router><Provider store={store}><Home /></Provider></Router>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
