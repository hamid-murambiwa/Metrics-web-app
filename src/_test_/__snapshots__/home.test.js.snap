// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render components render HomePage 1`] = `
<div
  className="art-con"
>
  <header>
    <img
      alt=""
      className="headerImg"
      src="97691843-group-of-dog-breeds-holding-bone-front-view-pet-background-banner.webp"
    />
    <div>
      <h1
        className="h-text"
      >
        Dog breeds
      </h1>
      <p>
        172 breeds
      </p>
    </div>
  </header>
  <div
    className="search-con"
  >
    <input
      id="search"
      onChange={[Function]}
      placeholder="Search for a breed"
      type="text"
    />
  </div>
  <div
    id="card-con"
  >
    <a
      className="card"
      href="/kjvgjk"
      onClick={[Function]}
    >
      <div>
        <div
          className="card_imgs"
        >
          <div
            className="ci"
          >
            <img
              className="d-img"
            />
            <img
              alt=""
              className="f-btn"
              src="icons8-forward-button-100.png"
            />
          </div>
        </div>
        <article
          className="title"
        >
          <h1 />
          <i>
            Life Span:
             
          </i>
        </article>
      </div>
    </a>
  </div>
</div>
`;
